skyportal/handlers/api/photometry.py

```diff
@@ -1327,6 +1339,11 @@ def put(self):
 
         refresh = self.get_query_argument('refresh', default=False)
 
+        if refresh is not None and str(refresh).lower() in ['true', 't', '1']:
```

Comment:
Are we missing a doc update to the PUT handler?

Reply:
added